src/components/EmojisRate/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import "./emojisRate.scss";
+import add from "./images/add.png";
```

Comment:
I think both require almost same resources to be charged, and the real difference is in the use, in this case I think we should add the icons in SVG(because they are not images, they are buttons) but I added the .png because they were more "easy" to implement and in this case we do not even have a real deployment with storage to have them :v, in the future I could try to see how to modify the SVG's colors (wrote down)

Reply:
**SVG**: These are vectors and it uses an algorithm to reorganized their pixels when their size is changed, these are the best choice to use in icons and some kinds of logos.
**PNG**: These are images that can handle 16 million colors, these are the best choice to use in images with high quality, this format is much larger than SVG.
To handle icons will be always better to use SVG, however, I don't know if using some icons with PNG format could affect our small application, but the truth is that the best practice would be using icons with SVG format.

[Difference between these formats by Adobe
](https://www.adobe.com/creativecloud/file-types/image/comparison/png-vs-svg.html#:~:text=While%20PNGs%20are%20capable%20of,size%20without%20losing%20their%20resolution.)

Reply:
@Hai5edfm @AreYouIvan Thank you guys. My question was more related with when is better include either images (SVG, PNG that you explained amazing) or import icons from an external source? I know that it won't have an important impact in our app, it's only to know how that works in bigger projects.

Reply:
Import the icons from an external source absolutely, the best practice would be to avoid sending binaries as icons/images to our repo. 😄